src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplatePhrase/PhrasalTemplatePhrase.tsx

```diff
@@ -31,26 +32,37 @@ import {
 import { PhrasalTemplatePhraseProps } from './PhrasalTemplatePhrase.types';
 import { getFieldPlaceholders, getNewDefaultField } from '../PhrasalTemplate.utils';
 import { KEYWORDS } from '../PhrasalTemplateField/PhrasalTemplateField.const';
-
+import { PhrasalTemplateResponseValuePhraseSchema } from 'modules/Builder/pages/BuilderApplet/BuilderApplet.schema';
 export const PhrasalTemplatePhrase = ({
   name = '',
   responseOptions = [],
   index = 0,
   onRemovePhrase,
+  field,
 }: PhrasalTemplatePhraseProps) => {
   const { t } = useTranslation('app');
+  // Use `useEffect` to trigger validation when `fields` or form state changes
```

Comment:
This inline comment doesn't seem to be applicable to the line that follows it; I'll move it somewhere more meaningful.